google-cloud-video-live_stream/samples/acceptance/helper.rb

```diff
@@ -19,33 +19,146 @@
 require "google/cloud/video/live_stream"
 
 require_relative "../../../.toys/.lib/sample_loader"
+require_relative "channel_definition"
+require_relative "input_definition"
+
+DELETION_THRESHOLD_TIME_HOURS_IN_KILOSECONDS = 10_800_000
```

Comment:
I assume you meant to name this MILLISECONDS. `10_800_000` kiloseconds is 10 billion seconds, or a little over 300 years, rather than 3 hours which is what the code is actually implementing.

Reply:
Done